src/reservation.js

```diff
@@ -0,0 +1,44 @@
+const reservation = (movie) => `
+<div class="modal modal-reservation" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      
+      <div class="modal-body">
+          <div>
+          <img src="https://image.tmdb.org/t/p/w500${movie.poster_path}" class="img-reservation">
+          </div>
+          <div class="modal-header">
+              <h5 class="modal-title">${movie.original_title}</h5>
+          </div>
```

Comment:
- [ ] **REQUIRED:** The reservation must throw an error if the selected date is reverse chronological. Users shouldn't not be able to select a second date which is before the first date.
![image](https://user-images.githubusercontent.com/2903437/147375059-569a45df-2017-4291-8257-eb6294867c08.png)